### PR TITLE
openCASCADE changes orientation according to the flag

### PR DESCRIPTION
**Pull request by ESeNonFossiIo**

### CODE REVIEW

File: source/opencascade/utilities.cc [hunk 1]
```diff
@@ -600,6 +600,13 @@ namespace OpenCASCADE
     Assert(props.IsCurvatureDefined(), ExcMessage("Curvature is not well defined!"));
     Standard_Real Mean_Curvature = props.MeanCurvature();
     Point<3> normal = Point<3>(Normal.X(),Normal.Y(),Normal.Z());
+
+    if (face.Orientation()==TopAbs_REVERSED)
+      {
+        normal *= -1;
+        Mean_Curvature *= -1;
+      }
```
Comment (tamiko) [inline source/opencascade/utilities.cc]: Do you mind to annotate this fix with a small comment and also provide a small entry in ```doc/news/changes.h```? :-)

Comment (ESeNonFossiIo) [inline source/opencascade/utilities.cc]: @tamiko. Is that as it should be?

Comment (bangerth) [inline source/opencascade/utilities.cc]: Is "verse" the "sense" in the comment? I don't recognize the word...

Comment (ESeNonFossiIo) [inline source/opencascade/utilities.cc]: sorry... my italinglish...

File: doc/news/changes.h [hunk 2]
```diff
@@ -103,6 +103,12 @@ inconvenience this causes.
  (Spencer Patty, 2016/02/11)
  </li>
 
+ <li> Fixed: The OpenCASCADE::push_forward_and_differential_forms()
+ function is now able to change the direction of the normal vector
+ according to Orientation() method.  
+ <br> 
+ (Mauro Bardelloni, 2016/03/02) 
+ </li>
```
Comment (luca-heltai) [inline doc/news/changes.h]: This should go on top. The changes are ordered with newest first.

Comment (ESeNonFossiIo) [inline doc/news/changes.h]: done

**Comment by bangerth** [comment]
I suppose it would be nice to eventually have a testcase for this. Maybe we can make one of the things you two were working on this week into a testcase since we already use it for the paper.

**Pull request merged**